tests/test_modeling_common.py

```diff
@@ -4308,6 +4308,44 @@ def _get_custom_4d_mask_test_data(self):
 
         return input_ids, position_ids, input_ids_shared_prefix, mask_shared_prefix, position_ids_shared_prefix
 
+    def test_sliding_window_mask(self):
```

Comment:
```suggestion
    def test_sliding_window_mask(self):
        """ Tests that we can control the sliding window attention behavior of a model. """
```
(So we can quickly understand what this test should be doing. Useful when refactoring multiple tests.)